README.md

```diff
@@ -346,6 +346,7 @@ For more Community Modules not listed here please see the [Terraform Module Regi
 - [terraformize](https://github.com/naorlivne/terraformize) - Apply\Destroy Terraform modules via a simple REST API endpoint.
 - [Terraform-Visual](https://github.com/hieven/terraform-visual) A simple but powerful tool to visualize Terraform plan.
 - [terragrunt](https://github.com/gruntwork-io/terragrunt) - Terragrunt is a thin wrapper for Terraform that provides extra tools for keeping your Terraform configurations DRY, working with multiple Terraform modules, and managing remote state.
+- [Terrahaxs](https://terrahaxs.com) - A GitOps Terraform CI/CD GitHub Application :heavy_dollar_sign:
```

Comment:
<img width="684" alt="CleanShot 2023-02-15 at 13 23 04" src="https://user-images.githubusercontent.com/52489/219131325-cfba6f88-9b0c-4b8a-af42-a9fe05a10d92.png">

Reply:
https://www.terrahaxs.com/ works though (with the www)